experiment/selinux/src/selinux.rs

```diff
@@ -91,6 +91,8 @@ pub enum SELinuxError {
     GetConfigKey(String),
     #[error("Invalid format for SELinux label: {0}")]
     InvalidSELinuxLabel(String),
+    #[error("Failed to run: {0}")]
+    Run(String),
```

Comment:
Hey one more question here - If we are introducing `Run` error just for the `main.rs` part, and not using it anywhere else, is it really worth adding an error type for it? My reasoning is `main.rs` is not going to be actually used for anything, I believe it is  just for testing and verifying the code, so we can either ignore the error there or unwrap if it is a fatal error. Correct me if I'm wrong.

Reply:
SGTM, removed `Run` error